src/app/pages/HomePage/HomePage.tsx

```diff
@@ -24,12 +25,39 @@ interface HomePageProps {
 }
 
 const HomePage = ({ pageData }: HomePageProps) => {
-  const { curations, description } = pageData;
-
-  const { translations, product, serviceLocalizedName, frontPageTitle, lang } =
-    useContext(ServiceContext);
+  const {
+    translations,
+    product,
+    serviceLocalizedName,
+    frontPageTitle,
+    lang,
+    brandName,
+  } = useContext(ServiceContext);
   const { topStoriesTitle, home } = translations;
+  const { title, description, curations } = pageData;
 
+  const itemListElement = curations
+    .map(({ summaries }) =>
+      summaries.map(summary => ({
+        '@context': 'http://schema.org',
+        '@type': 'ListItem',
+        url: summary.link,
+      })),
+    )
+    .flat()
+    .map((listItem, index) => {
+      return {
+        ...listItem,
+        position: index + 1,
+      };
+    });
+  const itemList = {
```

Comment:
Let's add a whitespace here 
```suggestion
    });

  const itemList = {
```